README.md

```diff
@@ -1,3 +1,11 @@
+> **Warning**
+> This repository is no longer maintained and will soon be archived.
+>
+> After a careful consideration we have made a difficult decision to no longer maintain this project.
+> Our efforts are concentrated in different areas of our scaling business and, unfortunately, it meant that we could not provide the love and care that this project deserves.
+> We are aiming to archive the project by 30/09/2022.
```

Comment:
Thanks @sprejjs for this PR.  Should we allow more time? Gut feel, this is a bit short: 2 months

Reply:
Hi Raz,

Good call. Extended till the end of the year.